src/Npgsql/NpgsqlCommand.cs

```diff
@@ -611,14 +609,14 @@ async Task PrepareLong()
                         {
                             if (pStatement.StatementBeingReplaced != null)
                             {
-                                Expect<CloseCompletedMessage>(await connector.ReadMessage(async), connector);
+                                Expect<CloseCompletedMessage>(await connector.ReadMessage(async, cancellationToken: cancellationToken), connector);
                                 pStatement.StatementBeingReplaced.CompleteUnprepare();
                                 pStatement.StatementBeingReplaced = null;
                             }
 
-                            Expect<ParseCompleteMessage>(await connector.ReadMessage(async), connector);
-                            Expect<ParameterDescriptionMessage>(await connector.ReadMessage(async), connector);
-                            var msg = await connector.ReadMessage(async);
+                            Expect<ParseCompleteMessage>(await connector.ReadMessage(async, cancellationToken: cancellationToken), connector);
```

Comment:
If the standard way of calling the method is with these two parameters, we can simply introduce another overload which would obviate needing to specify `cancellationtoken:` all the time.